<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OCR_MainWindow</class>
 <widget class="QMainWindow" name="OCR_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QWidget" name="image_frame" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="image_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>图片</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="exeEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_setExe">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="imageDirEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_set_ImageDir">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="resultEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_set_textDir">
            <property name="text">
             <string>设置</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_depatchTask">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>开始OCR识别</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="ocrTextResultBrowser"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOCR">
    <property name="title">
     <string>OCR配置设置</string>
    </property>
    <addaction name="action_setTesseractPath"/>
    <addaction name="action_setImageSaveDirPath"/>
    <addaction name="action_setResultTxtDir"/>
    <addaction name="separator"/>
    <addaction name="action_viewConfig"/>
    <addaction name="action_setDefaultConfigAll"/>
   </widget>
   <widget class="QMenu" name="menuOCR_2">
    <property name="title">
     <string>OCR语言设置</string>
    </property>
    <addaction name="action_setChinese"/>
    <addaction name="action_setEnglish"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>清理...</string>
    </property>
    <addaction name="action_clearImages"/>
    <addaction name="action_clearRes"/>
    <addaction name="action_clearAll"/>
   </widget>
   <addaction name="menuOCR"/>
   <addaction name="menuOCR_2"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_setChinese">
   <property name="text">
    <string>中文</string>
   </property>
  </action>
  <action name="action_setEnglish">
   <property name="text">
    <string>英文</string>
   </property>
  </action>
  <action name="action_setTesseractPath">
   <property name="text">
    <string>设置Tesseract可执行文件路径</string>
   </property>
  </action>
  <action name="action_setImageSaveDirPath">
   <property name="text">
    <string>设置图片保存路径</string>
   </property>
  </action>
  <action name="action_setResultTxtDir">
   <property name="text">
    <string>设置结果文件路径</string>
   </property>
  </action>
  <action name="action_viewConfig">
   <property name="text">
    <string>查看OCR配置</string>
   </property>
  </action>
  <action name="action_setDefaultConfigAll">
   <property name="text">
    <string>恢复所有的默认配置</string>
   </property>
  </action>
  <action name="action_clearImages">
   <property name="text">
    <string>清理图片</string>
   </property>
  </action>
  <action name="action_clearRes">
   <property name="text">
    <string>清理识别结果</string>
   </property>
  </action>
  <action name="action_clearAll">
   <property name="text">
    <string>清理全部</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
